app/models/mixins/retirement_mixin.rb

```diff
@@ -253,12 +253,12 @@ def valid_zone?
   end
 
   def system_context_requester
-    if try(:evm_owner_id).present?
-      User.find(evm_owner_id)
-    else
-      $log.info("System context defaulting to admin user because owner of #{name} not set.")
-      User.super_admin
+    if evm_owner.blank?
```

Comment:
Does everything `respond_to?(evm_owner)`?

Reply:
It comes from the OwnershipMixin, which is currently on all objects we retire.

Reply:
Hahaha, proved wrong by the specs. Everything except LoadBalancer. I've just no clue why we think you can retire them, I hope we don't, and I'm not sure if LoadBalancers should have the ownership mixin.

Reply:
@d-m-u Let's remove LoadBalancers from the retirement check code.

Reply:
@tinaafitz that's here: https://github.com/d-m-u/manageiq/commit/38eb29b29355f0d988ee8773751a7391e87899d1

Reply:
@d-m-u Great, thanks.